app/components/form-scheduling/template.hbs

```diff
@@ -24,43 +29,51 @@
       <label>{{radio-button selection=isRequestedHost value=false}}
         {{#if isService}}
           {{#if isGlobal}}
-            Automatically run on every host matching scheduling rules:
+            {{t 'formScheduling.autoRun'}}
           {{else}}
-            Automatically pick hosts for each {{if isVm 'virtual machine' 'container'}} matching scheduling rules:
+            {{#if (eq isVm 'virtual machines')}}
+              {{t 'formScheduling.autoPick.vm'}}
+            {{else}}
+              {{t 'formScheduling.autoPick.containers'}}
+            {{/if~}}
           {{/if}}
         {{else}}
-          Automatically pick a host matching scheduling rules:
+          {{t 'formScheduling.autoPick.host'}}
         {{/if}}
       </label>
     </div>
   {{else}}
     {{#if isService}}
       {{#if isGlobal}}
-        Automatically run on every host matching scheduling rules:
+        {{t 'formScheduling.autoRun'}}
       {{else}}
-        Automatically pick hosts for each {{if isVm 'virtual machine' 'container'}} matching scheduling rules:
+        {{#if (eq isVm 'virtual machines')}}
```

Comment:
This is not an equivalent if statement, the old one says "if isVm is true, then 'virtual machine'.  Otherwise, 'container'"  The new one says "if isVm has the value 'virtual machines', ..."  Change to `{{#if isVm}}`, or the whole if/else/endif block could be `{{t (if isVm 'formScheduling.autoPick.vm' 'formScheduling.autoPick.containers')}}`